packages/@react-stately/color/src/Color.ts

```diff
@@ -214,9 +214,9 @@ class RGBColor extends Color {
     }
 
     return new HSLColor(
-      hue * 360,
-      saturation * 100,
-      lightness * 100,
+      toFixedNumber(hue * 360, 2),
+      toFixedNumber(saturation * 100, 2),
+      toFixedNumber(lightness * 100, 2),
```

Comment:
Do we  also need to make this change to `HSBColor.toHSL()` and `HSLColor.toHSB()`